### PR TITLE
Bring back disruption for aggregation

### PR DESCRIPTION
**Pull request by deads2k**
This also gathers all percentiles for future evaluation of percentile rank.

### CODE REVIEW

**Comment by deads2k** [comment]
/hold

I need to prove working-ness

**Comment by deads2k** [comment]
/hold cancel

Seems to work

```
CGO_ENABLED=0 go build github.com/openshift/ci-tools/cmd/job-run-aggregator && ./job-run-aggregator analyze-job-runs --google-service-account-credential-file='/home/deads/openshift-ci-bigquery-credentials.json' --job=periodic-ci-openshift-release-master-ci-4.10-upgrade-from-stable-4.9-e2e-aws-ovn-upgrade --aggregation-id=f863f3c0-4b9c-11ec-acca-74d435fb2dc0 --explicit-gcs-prefix=logs/ovn-kubernetes-834-ci-4.10-upgrade-from-stable-4.9-e2e-aws-ovn-upgrade --job-start-time=2021-11-22T13:03:01Z

```

and 

```
CGO_ENABLED=0 go build github.com/openshift/ci-tools/cmd/job-run-aggregator && ./job-run-aggregator analyze-job-runs --google-service-account-credential-file=/home/deads/openshift-ci-bigquery-credentials.json --job=periodic-ci-openshift-release-master-ci-4.10-upgrade-from-stable-4.9-e2e-gcp-ovn-upgrade --payload-tag=4.10.0-0.nightly-2021-11-22-072055 --job-start-time=2021-11-22T07:00:00Z
```

File: pkg/jobrunaggregator/jobrunaggregatorapi/types.go [hunk 1]
```diff
@@ -35,7 +35,106 @@ type BackendDisruptionRow struct {
 }
 
 type BackendDisruptionStatisticsRow struct {
-	BackendName string
-	Mean        float64
-	P95         float64
+	BackendName       string
+	Mean              float64
+	StandardDeviation float64
+	P1                float64
```
Comment (stbenjam) [inline pkg/jobrunaggregator/jobrunaggregatorapi/types.go:41]: Nit: why not make this a slice of 100?

Comment (stbenjam) [reply pkg/jobrunaggregator/jobrunaggregatorapi/types.go:41]: I see, I guess it's easier to read into the struct.

**Comment by deads2k** [comment]
pushed image to quay for mirroring.  Let's see what tonight brings.

**Pull request merged**